libflux/flux-core/src/semantic/bootstrap.rs

```diff
@@ -96,23 +97,24 @@ pub struct FunctionDoc {
 }
 
 /// Implementation block for FunctionDoc to house future "FunctionDod" methods
-impl FunctionDoc {
-    /// Constructs a new FunctionDoc with the given name, headline, and flux type
-    fn new_with_args(
-        name: String,
-        headline: String,
-        description: String,
-        typ: String,
-    ) -> FunctionDoc {
-        FunctionDoc {
-            name,
-            headline,
-            description,
-            parameters: vec![],
-            flux_type: typ,
-        }
-    }
-}
+// impl FunctionDoc {
```

Comment:
If we don't need this code, let's go ahead and delete it.

Reply:
I have removed the unused code.